Prefer interfile or cross-file analysis (#1465)
semgrep/semgrep-docs: docs/semgrep-code/overview.md

* Customize how Semgrep Code scans your repositories
* Manage your users and facilitate team collaboration in remediating security issues

## OSS versus Semgrep Code analysis

By default, Semgrep Code can analyze interactions beyond a single function but within a single file, a process known as **cross-function or interprocedural analysis**. This smaller scope of analysis makes it faster and easier to integrate into developer workflows.

> Semgrep OSS can only analyze interactions within a single function, known as intraprocedural or single-function analysis. Semgrep Code scans take slightly longer to finish. <!-- can we have a stat for this -->

Reviewer: ```suggestion
Semgrep OSS can only analyze interactions within a single function, known as intraprocedural or single-function analysis. However, this means that Semgrep OSS is slightly faster than Semgrep Code. <!-- can we have a stat for this -->
```

Reworded this slightly, since this paragraph is focused on OSS.

Reviewer 2: I don't have a stat at the ready :( but need to work on this!

Semgrep Code also supports **[cross-file analysis](/semgrep-code/semgrep-pro-engine-intro/)** (interfile) analysis. These scans produce fewer false positives and more true positives, but take longer to complete.

## Enable Semgrep Code

1. Sign in to [<i class="fas fa-external-link fa-xs"></i> Semgrep Cloud Platform](https://semgrep.dev/login).
1. Click **[Settings](https://semgrep.dev/orgs/-/settings)**.
1. In the **Deployment** tab, click the **<i class="fa-solid fa-toggle-large-on"></i> Code (SAST)** toggle if it is not already enabled.

> Subsequent scans will now include Code scans.

Reviewer: ```suggestion
Subsequent scans will now include Semgrep Code scans.
```
